lib/mercury/wire_serializer.rb

```diff
@@ -23,17 +23,15 @@ def write_json(obj)
     end
 
     def read_json(bytes)
-      begin
-        Oj.compat_load(bytes, oj_options)
-      rescue Oj::ParseError => e
-        bytes
-      end
+      Oj.compat_load(bytes, oj_options)
+    rescue Oj::ParseError => e
+      bytes
```

Comment:
Removing the `begin` and having the `rescue`'s at the same indent level as `def` throw me for a loop. If we decide as a team to adhere to this rule, it will take some getting used to for me.